modules/react/pagination/stories/pagination.stories.mdx

```diff
@@ -91,50 +35,45 @@ yarn add @workday/canvas-kit-react
 
 ## Usage
 
-`Pagination` is a compound component, meaning it can be put together and used in a variety of
-different ways. We'll first cover [basic usage](#basic-usage) examples followed by
-[advanced usage](#advanced-usage). Each of the examples is intended to be standalone, so you can
-skip to the section that best fits your use case.
-
-These examples are designed to provide copy-and-paste snippets to help you get up and running
-quickly without much explanation. Further detail on individual component props, behavior, and
-accessibility can be found in the [Components](#components) section.
+### Basic Example
 
----
+`Pagination` includes a container `Pagination` component and a number of subcomponents which can be
+composed in a variety of ways.
 
-### Basic Usage
+In this example, we set up a basic `Pagination` component with the default range of 5 pages, as well
+as step controls (`Pagination.StepToPreviousButton` and `Pagination.StepToNextButton`) that allow
+you to move to the next page or the previous page.
 
-Below are examples we expect will work for most use cases. Please refer to the
-[Advanced Usage](#advanced-usage) section for details on supporting custom implementations.
+<ExampleCodeBlock code={Basic} />
 
-#### Step Controls
+### Hoisted Model
 
-In this example, the component uses step controls (`Pagination.StepToPreviousButton` and
-`Pagination.StepToNextButton`) that allow you to move to the next page and previous pages.
+By default, `Pagination` will create and use its own [model](#model) internally. Alternatively, you
+may configure your own model with `usePaginationModel` and pass it to `Pagination` via the `model`
+prop. This pattern is referred to as
+[hoisting the model](/getting-started/for-developers/resources/compound-components/#configuring-a-model)
+and provides direct access to its `state` and `events` outside of the `Pagination` component.
 
-<ExampleCodeBlock code={StepControls} />
+In this example, we set up external observation of the model state and create an external button to
+trigger an event to change the current page.
 
----
+<ExampleCodeBlock code={HoistedModel} />
 
-#### Jump Controls
+### Jump Controls
 
 This example adds jump controls (`Pagination.JumpToFirstButton` and `Pagination.JumpToLastButton`)
-that allow you to skip to the first and last items in the range.
+that allow you to skip to the first and last pages in the range.
 
 <ExampleCodeBlock code={JumpControls} />
 
----
-
-#### GoTo Form
+### GoTo Form
 
 This example adds a form (`Pagination.GoToForm`) that allows you to skip to a specific page within
 the range.
 
 <ExampleCodeBlock code={GoToForm} />
 
----
-
-#### Additional Details
+### Additional Details
```

Comment:
I'm guessing the `Additional Details` content should be included in the `Basic Example` section given that it adds a required `aria-live` region?

Reply:
Yes. That's correct. It should be in every example. 👍🏻